src/components/steps/step.jsx

```diff
@@ -0,0 +1,30 @@
+const PropTypes = require('prop-types');
+const React = require('react');
+
+const FlexRow = require('../../components/flex-row/flex-row.jsx');
+
+require('./steps.scss');
+
+const Step = props => (
```

Comment:
This could get confusing because we already have components called `Step` and `Steps` in `registration`...